NuGetTest/src/Modules/Lombiq.OSOCE.NuGet.TestModule/Controllers/AnalyzersTestController.cs

```diff
@@ -8,9 +8,9 @@ public class AnalyzersTestController : Controller
 {
     private readonly IClock _clock;
 
-    // Since the Analyzers NuGet package is added, you can see that it works like this: Disable the IDE0021 rule in the
-    // root Analyzers OrchardCore.ruleset file, then change this constructor to use a block body. You should get an
-    // analyzer violation since the expression-bodied rule is still enforced from the package
+    // Since the Analyzers NuGet package is added, you can see that it works like this: change this constructor to use a
+    // block body. You should get an analyzer violation since the expression-bodied rule is still enforced from the
+    // package
```

Comment:
```suggestion
    // Since the Analyzers NuGet package is added, you can see that it works like this: Change this constructor to use a
    // block body. You should get an analyzer violation since the expression-bodied rule is still enforced from the
    // package.
```

Mr. Z insists on Capitalization after the colon, so please adhere. I insist on a dot at the end of the sentence.